frontend/src/components/AccountReceiveTable.vue

```diff
@@ -284,7 +341,8 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
   const mainTableColumns = [
     { align: 'left', field: 'block', label: 'Date Received', name: 'date', sortable: true, sort: sortByTime },
     { align: 'left', field: 'amount', label: 'Amount', name: 'amount', sortable: true, format: toString },
-    { align: 'left', field: 'receipt', label: 'From', name: 'from', sortable: true },
+    { align: 'left', field: 'receipt', label: 'Sender', name: 'from', sortable: true },
+    { align: 'left', field: 'receiver', label: 'Stealth Receiver', name: 'receiver', sortable: true },
```

Comment:
Like @apbendi mentioned, let's flip `sortable` to `false` here. Sorting on randomly generated stealth addresses doesn't add value so that will resolve the "clicking tooltip also sorts" problem